packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -96,6 +96,22 @@ contract L2Genesis is Deployer {
         0x9DCCe783B6464611f38631e6C851bf441907c710 // 29
     ];
 
+    /// @notice The address of the deployer account.
+    address internal deployer;
+
+    /// @notice Sets up the script and ensures the deployer account is used to make calls.
+    function setUp() public override {
+        deployer = makeAddr("deployer");
+        super.setUp();
+    }
+
+    /// @notice Modifier that starts and ends a prank around the function call.
+    modifier prank(address _addr) {
+        vm.startPrank(_addr);
+        _;
+        vm.stopPrank();
+    }
```

Comment:
Looks like we only use this modifier once, why not just inline the start/stopPrank calls?